### ISSUE TITLE
mlp_mixer/lucidrains/ pytorch model hangs in CI in tt-xla

### ISSUE THREAD

**Issue by ctr-pmuruganTT** [tenstorrent/tt-mlir#5601]
`mlp_mixer/lucidrains/pytorch-base-single_device-full-inference ` model started hanging in CI [nightly](https://github.com/tenstorrent/tt-xla/actions/runs/18733729390/job/53436218189) on 22/10/2025

After bisection, the errors seems to have come after this tt-mlir commit:
91779dfa5a66e619af0cc12310600315f9c462f8

Attaching the logs 

before this commit :  [lucid_mlp_mlir_7_e20cd5f3f.log](https://github.com/user-attachments/files/23238460/lucid_mlp_mlir_7_e20cd5f3f.log)

In this commit :  [lucid_mlp_mlir_5_91779dfa5.log](https://github.com/user-attachments/files/23238464/lucid_mlp_mlir_5_91779dfa5.log)

Ref Issue created in xla : https://github.com/tenstorrent/tt-xla/issues/1791

**Comment by ctr-pmuruganTT** [comment]
Tagging @pglusacTT as the owner of the commit `Add batch norm training and update batch norm`

**Comment by pglusacTT** [comment]
This is a metal bug that is actively worked on, here is the PR: https://github.com/tenstorrent/tt-metal/pull/31511

**Comment by sgligorijevicTT** [comment]
Underlying metal issue is fixed, xla model is now passing(unksipping is on pr), closing
